### PR TITLE
Implement UI for channel page

### LINKED ISSUE #26: As a logged in user, I would like to view a channel page

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### What are the relevant tickets?
Part of #26 

#### What's this PR do?
Adds the channel listing page.

#### How should this be manually tested?
Go to /channels/something and verify you see the page as shown in the screenshot.
Also go to `/api/v0/channels` and `/api/v0/channels/pics` and verify you see `public_description` in there now. it will probably be empty strings unless you've set it on a sub.

#### Screenshots (if appropriate)
selection_077

### CODE REVIEW

**Comment by aliceriot** [comment]
If I go to `/channels/foobar` or `/channels/test_channel` I just see a blank page. It looks like the `Route` component is rendering `null` for it's children somehow?

nullnullwhy

**Comment by aliceriot** [comment]
I also don't seem to be able to visit the API pages (like `/api/v0/channels/pics`), I just get an empty React app at that URL.

**Comment by rhysyngsun** [comment]
@aliceriot I've fixed those. One was a regression with how I dealt with urls.py, the other was the new `<Loading/>` I wrote in this PR was expecting the `posts` reducer to set `loaded = true` which it never did. I removed that from the list for now.

**Comment by aliceriot** [comment]
API stuff appears to be working! although we should probably note down the quirk that new users have to take some sort of 'action' (subscribing to a subreddit worked for me) before their subscriptions show up in the API.

**Comment by aliceriot** [comment]
UI shows up too

**Comment by rhysyngsun** [comment]
@aliceriot I'll file a bug for the subreddit list not showing up right away. Might be some user initialization that we need to add to the refresh_token api. CC @noisecapella

File: static/scss/sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+
```
Comment (aliceriot) [inline static/scss/sidebar.scss]: blank lines

Comment (rhysyngsun) [inline static/scss/sidebar.scss]: :trollface:

**Pull request merged**